spec/src/main/asciidoc/architecture.adoc

```diff
@@ -142,6 +142,9 @@ export MP_METRICS_TAGS=app=shop,tier=integration,special=deli\=ver\,y
 
 Global tags and tags registered with the metric are included in the output returned from the REST API.
 
+Global tags MUST NOT be automatically added to the `MetricID` objects, but they must be appended to all metrics in a way that
```

Comment:
suggest:
Global tags MUST NOT be added to the `MetricID` objects.  Global tags must be included in list of tags when metrics are exported.